week2/Program.cs

```diff
@@ -0,0 +1,117 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+// Class excercises:
+app.MapGet("/excercise1", (string a, string b, string operation) =>
+{
+    var IsaValid = int.TryParse(a, out var parseda);
+    var IsbValid = int.TryParse(b, out var parsedb);
+    
+    if(!IsaValid || !IsbValid)
+    {
+        return "Invalid inputs";
```

Comment:
If you want to have nicer return you can use `Results.BadRequest()` and `Results.Ok()`